playground/backend/internal/executors/executor_builder.go

```diff
@@ -82,87 +82,97 @@ func (b *ExecutorBuilder) WithTestRunner() *UnitTestExecutorBuilder {
 	return &UnitTestExecutorBuilder{*b}
 }
 
-//WithCommand adds compile command to executor
+// WithCommand adds compile command to executor
 func (b *CompileBuilder) WithCommand(compileCmd string) *CompileBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		e.compileArgs.commandName = compileCmd
 	})
 	return b
 }
 
-//WithWorkingDir adds dir path to executor
+// WithWorkingDir adds dir path to executor
 func (b *CompileBuilder) WithWorkingDir(dir string) *CompileBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		e.compileArgs.workingDir = dir
 	})
 	return b
 }
 
-//WithArgs adds compile args to executor
+// WithArgs adds compile args to executor
 func (b *CompileBuilder) WithArgs(compileArgs []string) *CompileBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		e.compileArgs.commandArgs = compileArgs
 	})
 	return b
 }
 
-//WithFileName adds file name to executor
+// WithFileName adds file name to executor
 func (b *CompileBuilder) WithFileName(fileName string) *CompileBuilder {
+	return b.WithFileNames([]string{fileName})
+}
+
+// WithFileNames adds file names to executor
+func (b *CompileBuilder) WithFileNames(fileNames []string) *CompileBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
-		e.compileArgs.fileName = fileName
+		e.compileArgs.fileNames = fileNames
 	})
 	return b
 }
 
-//WithExecutableFileName adds file name to executor
+// WithExecutableFileName adds file name to executor
 func (b *RunBuilder) WithExecutableFileName(name string) *RunBuilder {
+	return b.WithExecutableFileNames([]string{name})
+}
+
+// WithExecutableFileNames adds file name to executor
+func (b *RunBuilder) WithExecutableFileNames(names []string) *RunBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
-		e.runArgs.fileName = name
+		e.runArgs.fileNames = names
 	})
 	return b
 }
 
-//WithWorkingDir adds dir path to executor
+// WithWorkingDir adds dir path to executor
 func (b *RunBuilder) WithWorkingDir(dir string) *RunBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		e.runArgs.workingDir = dir
 	})
 	return b
 }
 
-//WithCommand adds run command to executor
+// WithCommand adds run command to executor
 func (b *RunBuilder) WithCommand(runCmd string) *RunBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		e.runArgs.commandName = runCmd
 	})
 	return b
 }
 
-//WithArgs adds run args to executor
+// WithArgs adds run args to executor
 func (b *RunBuilder) WithArgs(runArgs []string) *RunBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		e.runArgs.commandArgs = runArgs
 	})
 	return b
 }
 
-//WithGraphOutput adds the need of graph output to executor
+// WithGraphOutput adds the need of graph output to executor
 func (b *RunBuilder) WithGraphOutput() *RunBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		//todo
```

Comment:
What is the work of the `//todo`?  Could we instead refer to a GitHub issue?

Reply:
This functions are not used anywhere and we weren't able to find any tasks referring to them. Removed both of them.